parser-typechecker/src/Unison/Runtime/Builtin.hs

```diff
@@ -1582,191 +1607,191 @@ declareForeigns = do
             _ -> ReadWriteMode
       in openFile fname mode
 
-  declareForeign "IO.closeFile.impl.v3" boxToEF0 $ mkForeignIOF hClose
-  declareForeign "IO.isFileEOF.impl.v3" boxToEFBool $ mkForeignIOF hIsEOF
-  declareForeign "IO.isFileOpen.impl.v3" boxToEFBool $ mkForeignIOF hIsOpen
-  declareForeign "IO.isSeekable.impl.v3" boxToEFBool $ mkForeignIOF hIsSeekable
+  declareForeign Untracked "IO.closeFile.impl.v3" boxToEF0 $ mkForeignIOF hClose
+  declareForeign Untracked "IO.isFileEOF.impl.v3" boxToEFBool $ mkForeignIOF hIsEOF
+  declareForeign Untracked "IO.isFileOpen.impl.v3" boxToEFBool $ mkForeignIOF hIsOpen
+  declareForeign Untracked "IO.isSeekable.impl.v3" boxToEFBool $ mkForeignIOF hIsSeekable
```

Comment:
@dolio I'd make all these `Tracked`. Basically any builtin that starts with `IO` should be `Tracked`. The user can always add more things to the sandbox, but they have no way to remove things from the sandbox that are `Untracked`. 

The only thing the user should be able to do with an empty sandbox is pure computation. They could write an infinite loop or consume all the RAM on the machine, but they shouldn't be able to learn anything about the file system, network sockets, etc.

Reply:
Do you care about `Handle/Socket.toText`? That just prints out some kind of information string about the value (not sure what exactly it prints out).